### PR TITLE
Ocra bigdl example: Attention

### PR DESCRIPTION
**Pull request by leonardozcm**
Tested on local and yarn:
```
2021-01-27 13:41:06 INFO  DistriOptimizer$:427 - [Epoch 1 24320/25000][Iteration 190][Wall Clock 1128.313711744s] Trained 128.0 records in 6.101200233 seconds. Throughput is 20.979479 records/second. Loss is 0.21225765.
2021-01-27 13:41:12 INFO  DistriOptimizer$:427 - [Epoch 1 24448/25000][Iteration 191][Wall Clock 1133.60985568s] Trained 128.0 records in 5.296143936 seconds. Throughput is 24.168528 records/second. Loss is 0.1766904.
2021-01-27 13:41:18 INFO  DistriOptimizer$:427 - [Epoch 1 24576/25000][Iteration 192][Wall Clock 1139.36616416s] Trained 128.0 records in 5.75630848 seconds. Throughput is 22.236473 records/second. Loss is 0.24468824.
2021-01-27 13:41:23 INFO  DistriOptimizer$:427 - [Epoch 1 24704/25000][Iteration 193][Wall Clock 1145.075981451s] Trained 128.0 records in 5.709817291 seconds. Throughput is 22.41753 records/second. Loss is 0.16636617.
2021-01-27 13:41:29 INFO  DistriOptimizer$:427 - [Epoch 1 24832/25000][Iteration 194][Wall Clock 1150.685762949s] Trained 128.0 records in 5.609781498 seconds. Throughput is 22.81729 records/second. Loss is 0.15286463.
2021-01-27 13:41:34 INFO  DistriOptimizer$:427 - [Epoch 1 24960/25000][Iteration 195][Wall Clock 1156.17795664s] Trained 128.0 records in 5.492193691 seconds. Throughput is 23.305805 records/second. Loss is 0.16795565.
2021-01-27 13:41:39 INFO  DistriOptimizer$:427 - [Epoch 1 25088/25000][Iteration 196][Wall Clock 1161.351888668s] Trained 128.0 records in 5.173932028 seconds. Throughput is 24.739405 records/second. Loss is 0.151119.
2021-01-27 13:41:39 INFO  DistriOptimizer$:472 - [Epoch 1 25088/25000][Iteration 196][Wall Clock 1161.351888668s] Epoch finished. Wall clock time is 1161527.104629 ms
```

### CODE REVIEW

File: pyzoo/zoo/examples/orca/learn/bigdl/attention/transformer.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                                 "spark.driver.extraJavaOptions": "-Xss512m"}
+                           )
+elif cluster_mode == "yarn":
+    sc = init_orca_context(cluster_mode="yarn-client", num_nodes=8, cores=8,
+                           driver_memory="20g",
+                           memory="100g",
+                           conf={"spark.executor.extraJavaOptions": "-Xss512m",
+                                 "spark.driver.extraJavaOptions": "-Xss512m"}
+                           )
+
```
Comment (yangw1234) [inline pyzoo/zoo/examples/orca/learn/bigdl/attention/transformer.py:51]: what if a user passes a wrong `cluster_mode`

**Comment by yangw1234** [comment]
add the example to jenkins?

**Comment by leonardozcm** [comment]
jenkins script added

**Comment by yangw1234** [comment]
LGTM

**Comment by leonardozcm** [comment]
jenkins passed @yangw1234 
http://10.239.47.210:18888/job/ZOO-PR-Validation/5017/
http://10.239.47.210:18888/job/ZOO-PR-Python-ExampleTests/2222/
http://10.239.47.210:18888/job/ZOO-PR-Python-integration-test/47/

**Comment by jason-dai** [comment]
remove the original example `pyzoo/zoo/examples/attention`

**Comment by leonardozcm** [comment]
> remove the original example `pyzoo/zoo/examples/attention`

OK

**Pull request merged**